tsconfig.base.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "module": "CommonJS",
+    "module": "NodeNext",
```

Comment:
I recommend not modifying the output here, as it might lead to unpredictable behavior.

Reply:
since CI passed, let's merge it and if we find problem we can revert this change